### DISCUSSION TITLE
Cannot connect to the cockpit GUI through browser, port 64294/64297

### DISCUSSION THREAD

**Discussion by onlypwns** [telekom-security/tpotce#1089] [Q&A] [answered]
🗨️ Please post your questions in [Discussions](https://github.com/telekom-security/tpotce/discussions) and keep the issues for **issues**. Thank you 😁.<br>

Before you post your issue make sure it has not been answered yet and provide `basic support information` if you come to the conclusion it is a new issue.

- 🔍 Use the [search function](https://github.com/dtag-dev-sec/tpotce/issues?utf8=%E2%9C%93&q=) first
- 🧐 Check our [WIKI](https://github.com/dtag-dev-sec/tpotce/wiki)
- 📚 Consult the documentation of 💻 [Debian](https://www.debian.org/doc/), 🐳 [Docker](https://docs.docker.com/), the 🦌 [ELK stack](https://www.elastic.co/guide/index.html) and the 🍯 [T-Pot Readme](https://github.com/dtag-dev-sec/tpotce/blob/master/README.md).
- **⚠️ Provide [basic support information](#info) or similar information with regard to your issue or we can not help you and will close the issue without further notice**

<br>
<br>
<br>

<a name="info"></a>
## ⚠️ Basic support information (commands are expected to run as `root`)

- What version of the OS are you currently using `lsb_release -a` and `uname -a`?

![image](https://user-images.githubusercontent.com/88723154/167746399-0eb213de-7789-4ea8-99a0-47bc1c17710a.png)

- What T-Pot version are you currently using?
>Latest version, just installed

- What edition (Standard, Nextgen, etc.) of T-Pot are you running?
>Standard

- What architecture are you running on (i.e. hardware, cloud, VM, etc.)?
>Azure Cloud

- Did you have any problems during the install? If yes, please attach `/install.log` `/install.err`.
>No

- How long has your installation been running?
>Maybe 10ish minutes

- Did you install upgrades, packages or use the update script?
I did use the update script

- Did you modify any scripts or configs? If yes, please attach the changes.
>No

- Please provide a screenshot of `glances` and `htop`.

>glances
![image](https://user-images.githubusercontent.com/88723154/167746686-4e8e9e82-93f3-4d6d-ac8e-759f27e538c1.png)

>htop
![image](https://user-images.githubusercontent.com/88723154/167747271-e7336865-4eea-48e8-a5d6-118f3e2edd8b.png)

- How much free disk space is available (`df -h`)?

![image](https://user-images.githubusercontent.com/88723154/167746790-399e4bbd-1ae1-4f1a-a88f-53360b24eb15.png)

- What is the current container status (`dps.sh`)?

![image](https://user-images.githubusercontent.com/88723154/167746903-57e8e1da-a9ba-4c54-812b-e2ca85026d48.png)

- What is the status of the T-Pot service (`systemctl status tpot`)?

![image](https://user-images.githubusercontent.com/88723154/167746948-fd7bed51-efb8-4ed5-9185-81df3be0fdc5.png)

- What ports are being occupied? Stop T-Pot `systemctl stop tpot` and run `netstat -tulpen`

![image](https://user-images.githubusercontent.com/88723154/167747092-487cd2fc-b99f-4d88-9ce5-6d6aabb63e8d.png)

- If a single container shows as `DOWN` you can run `docker logs <container-name>` for the latest log entries
>N/A

**Comment by t3chn0m4g3** [comment, marked as answer]
This is not an issue, but one of the limitations mentioned in the [Readme](https://github.com/telekom-security/tpotce#running-in-a-cloud).
The service for `cockpit, 64294` seems to be bound to IPv6 according to your `netstat -tulpen` output. The same is probably happening to T-Pot WebUI.
Please check other issues / discussions regarding an Azure setup or check if it is possible to disable IPv6 within Azure.
In the past T-Pot forced IPv4 in `/etc/sysctl.conf` ... :
```
net.ipv6.conf.all.disable_ipv6 = 1
net.ipv6.conf.default.disable_ipv6 = 1
net.ipv6.conf.lo.disable_ipv6 = 1
```
... but this may cause other problems.

**Reply by onlypwns** [reply]
Thanks for your reply. Tried changing the .conf file but did not work. At the end I've changed to Linode and works perfectly! Thank you so much
